### ISSUE TITLE
Add support for custom templates in multi select

### ISSUE THREAD

**Issue by brachi-wernick** [primefaces/primeng#3682]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[X ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
no support for custom templates in multi-select items

**Comment by Ketec** [comment]
Just end up with this problem aswell. Needing to differentiate different data sources wit ha icon or color.

**Comment by brachi-wernick** [comment]
closed, added in 4.2.0
